tuner/examples/punet/mmt.mlir

```diff
@@ -0,0 +1,76 @@
+// RUN: iree-compile --iree-hal-target-backends=rocm --iree-rocm-target-chip=gfx942 \
+// RUN:   --iree-rocm-link-bc=true --iree-rocm-bc-dir=/opt/rocm/amdgcn/bitcode \
+// RUN:   --iree-global-opt-propagate-transposes=true --iree-opt-outer-dim-concat=true \
+// RUN:   --iree-opt-const-eval=false --iree-codegen-gpu-native-math-precision=true --iree-rocm-waves-per-eu=2 \
+// RUN:   --iree-preprocessing-pass-pipeline='builtin.module(iree-preprocessing-transpose-convolution-pipeline)' \
+// RUN:   --iree-codegen-llvmgpu-use-vector-distribution --iree-codegen-transform-dialect-library=config.mlir \
+// RUN:   %s -o %s.vmfb
```

Comment:
I'd prefer for examples to be minimal. Could leave off all the optional flags here and just use `iree-compile  --iree-hal-target-backends=rocm --iree-rocm-target-chip=gfx942`